orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/OrcaMessageHandler.kt

```diff
@@ -52,6 +58,7 @@ internal interface OrcaMessageHandler<M : Message> : MessageHandler<M> {
         .taskById(taskId)
         .let { task ->
           if (task == null) {
+            log.error("InvalidTaskId: Unable to find task {} in existing tasks {} while processing message {}", taskId, stage.tasks.map { it.id }, this)
```

Comment:
Will `this` actually print a human readable representation of the message vs the object type?

Reply:
Good call, I'm not sure and I can't get orca to start to test it out. Changing to print the class name

Reply:
yes, since `this` is of type `Message` and since every message is a `data class` they should get a sensible `toString()` from the compiler